tools/cockpit.spec

```diff
@@ -167,6 +189,9 @@ cp src/bridge/polkit-workarounds.rules %{buildroot}/%{_datadir}/polkit-1/rules.d
 mkdir -p $RPM_BUILD_ROOT%{_sysconfdir}/pam.d
 install -p -m 644 %{SOURCE1} $RPM_BUILD_ROOT%{_sysconfdir}/pam.d/cockpit
 rm -f %{buildroot}/%{_libdir}/cockpit/*.so
+%if 0%{?suse_version}
+install -d %{buildroot}%{_docdir}/%{name}
```

Comment:
Right, but in this case we are assembling that directory manually, see the line below. I don't think it's driven by 'make install'.